docs/pages/guides/editing-richtext.mdx

```diff
@@ -0,0 +1,118 @@
+
+**aa**aa
+
+```jsx
+<TextInput>
+  <Text>
+    <Text style={{ fontWeight: 900 }}>aa</Text>aa
+  </Text>
+</TextInput>
+```
+
+And you append a fifth letter a, the position of your cursor should determine if the new letter is a part of the bold string or not. Unfortunately, the callback only returns `aaaaa`.
+
+There is an additional `onSelectionChange` prop that could be use to get that information, but it makes the task significantly harder. Inserting additional characters (such as newlines for lists or bullet points) also desynchronises the selection.
+
+There are some attempts to build such an editor, such as [markdown-editor](https://github.com/shakogegia/markdown-editor) but no widely used packages exist.
+
+## Markdown editors with always visible styling markers
+
+If you’re ok with using markdown to style text, you can render markdown in a separate, non-editable view while editing. Here’s a package that does that (https://github.com/kunall17/react-native-markdown-editor), but it’s also easy to build on your own using any markdown renderer.
+
+This is an editing experience more suited to power users or programming/tech applications.
+
+You can also explore rendering rich text while showing markdown only in the selected chunk of text or other hybrid approaches.
+
+## Native editors
+
+You can consider using a native Android/iOS rich text editor wrapped into a react-native module. There are a couple of options:
+
+- [react-native-aztec](https://github.com/WordPress/gutenberg/tree/trunk/packages/react-native-aztec)
+- [gutenberg-mobile](https://github.com/wordpress-mobile/gutenberg-mobile)
+
+You can also wrap any native rich text editor using [expo-modules](/modules/overview/), but if you use different ones on each platform you will need to unify their APIs and input formats somehow.
+
+> Rich text is usually represented using [abstract syntax trees](https://en.wikipedia.org/wiki/Abstract_syntax_tree).
+>
+> For example, a bullet list can be a node of type `bulleted-list` with several children of type `list-item`. You can convert both HTML and markdown to a suitable AST format.
+
+There is also an effort to port the lexical editor to iOS and Android and provide a react-native wrapper. You can track it here: https://github.com/facebook/lexical/discussions/2410.
+
+# Summary
+
+While there are a lot of great options for showing rich text, is no one size fits all solution for rich text editing in react-native. In fact, there's even no popular solution that is sufficient in most usecases. Further contribution from the community is needed to improve existing solutions or add new ones.
+
+For now, you need to make a careful choice between a more complex, native editor that offers more features but may be harder to maintain, or an editor built on top of react-native primitives. This depends on how core the feature is to your app, how many features you need in your editor, and how much effort are you willing to spend on building it out.
```

Comment:
```suggestion
While there are many great options for showing rich text, there is no one-size-fits-all solution for rich text editing in React Native. There's no popular solution that is sufficient in most use cases. Further contribution from the community is needed to improve existing solutions or add new ones.

For now, you need to carefully choose between a more complex, native editor that offers more features but may be harder to maintain or an editor built on top of react-native primitives. This depends on how core the feature is to your app, how many features you need in your editor, and how much effort you are willing to spend on building it out.
```

- Let's not overephasize the about not a common/most-used/de facto solution in the first paragraph's second line